pkg/api/api.go

```diff
@@ -240,6 +252,46 @@ func requestPostageBatchId(r *http.Request) ([]byte, error) {
 	return nil, errInvalidPostageBatch
 }
 
+type AuthKeyResponse struct {
+	Key string `json:"key"`
+}
+
+type RoleRequest struct {
+	Role string `json:"role"`
+}
```

Comment:
These types does not need to be exported, and can be provided through export_test.go for tests.